tests/components/smarttub/conftest.py

```diff
@@ -81,6 +81,16 @@ def mock_spa(spa_state):
 
     spa_state.lights = [mock_light_off, mock_light_on]
 
+    mock_cover_sensor = create_autospec(smarttub.SpaSensor, instance=True)
+    mock_cover_sensor.spa = mock_spa
+    mock_cover_sensor.address = "address1"
+    mock_cover_sensor.name = "{cover-sensor-1}"
```

Comment:
I think a good future improvement would be for the library to remove the `{}`, because I don't really see why we would ever need them

Reply:
Agree we should never need them. Perhaps the library could generate a human-readable name for us and not expose this at all

Reply:
In the vendor app, these sensor show up as singletons. There's either a cover sensor or none, a water sensor or none, etc. I thought we needed some kind of name here, to differentiate between the different binary sensors, but if not, let me know what would be preferable and I'll update